wormhole/wormhole.go

```diff
@@ -80,7 +80,16 @@ func (c *Client) relayURL() (*url.URL, error) {
 	if rurl == "" {
 		rurl = DefaultTransitRelayURL
 	}
-	return url.Parse(rurl)
+	var url, err = url.Parse(rurl)
+	if err != nil {
+		return nil, err
+	}
+	/* Backwards compatibility: Find old tcp:host style URLs and insert "//" then try again */
+	if url.Scheme == "tcp" && url.Opaque != "" {
+		rurl := "tcp://" + rurl[:4]
```

Comment:
Should be _rurl[4:]_ otherwise doesnt work with _tcp:localhost:4001_

Reply:
Oops ^^

Interestingly I thought I'd tried it out and the tests ran through just fine …

Reply:
All test contains tcp://